mediaPlayer/SuperMediaPlayer.cpp

```diff
@@ -4209,6 +4210,16 @@ void SuperMediaPlayer::SetOnRenderCallBack(onRenderFrame cb, void *userData)
     mFrameCb = cb;
     mFrameCbUserData = userData;
 }
+
+void SuperMediaPlayer::SetAudioRenderingCallBack(onRenderFrame cb, void *userData)
+{
+    mAudioRenderingCb = cb;
+    mAudioRenderingCbUserData = userData;
+    if (mAudioRender) {
```

Comment:
Can‘t update the callback here，it's not thread safe，SetAudioRenderingCallBack can only effect when set before calling prepare